common/src/main/java/com/mx/common/exception/PathRequestException.java

```diff
@@ -6,7 +6,22 @@
 import lombok.Getter;
 import lombok.Setter;
 
+import com.mx.common.accessors.AccessorException;
+import com.mx.common.accessors.AccessorMethodNotImplementedException;
+import com.mx.common.accessors.AccessorSystemException;
+import com.mx.common.accessors.AccessorUserException;
 import com.mx.common.accessors.PathResponseStatus;
+import com.mx.common.accessors.ResourceNotFoundException;
+import com.mx.common.accessors.UnauthorizedException;
+import com.mx.common.connect.CircuitOpenException;
+import com.mx.common.connect.ConnectException;
+import com.mx.common.connect.ServiceUnavailableException;
+import com.mx.common.connect.TimeoutException;
+import com.mx.common.connect.TooManyRequestsException;
+import com.mx.common.connect.UpstreamErrorException;
+import com.mx.common.facility.FacilityException;
+import com.mx.common.gateway.BehaviorException;
+import com.mx.common.gateway.GatewayException;
 
 /**
  * Base exception for any request-based error. These are errors that occur at any level during a request.
```

Comment:
If we change `ConnectException` to extend `PathRequestException` the structure will need to be updated here as well (in the Javadoc).

Reply:
Fixed